src/OpenSearch/RequestFactory.php

```diff
@@ -45,4 +45,23 @@ public function createRequest(
         return $request;
     }
 
+    /**
+     * Create a query string from an array of parameters.
+     */
+    private function createQuery(array $params): string
+    {
+        ksort($params);
```

Comment:
@kimpepper Is this by reference and does it modify the caller's $params? It's probably not desired.

Reply:
Ah yes. I can't remember why I added that. I've created a PR to remove #276